### PR TITLE
Fix link to Supabase Cloud Functions

### PR DESCRIPTION
**Pull request by sultanby**
According to https://supabase.com/docs/guides/functions/deploy link to deploy supabase edge functions has been changed from 
https://api.telegram.org/bot<BOT_TOKEN>/setWebhook?url=https://<PROJECT_NAME>.functions.supabase.co/telegram-bot?secret=<BOT_TOKEN> 
to 
https://api.telegram.org/bot<BOT_TOKEN>/setWebhook?url=https://<PROJECT_REFERENCE_ID>.supabase.co/functions/v1/telegram-bot?secret=<BOT_TOKEN>
Here is the fix for all languages (en, zh, uk, es, id) where outdated url was present.

### CODE REVIEW

**Comment by KnorpelSenf** [comment]
Thanks for the fix. How did you obtain the translations?

**Comment by sultanby** [comment]
@KnorpelSenf about uk i asked a friend, for es and id translations i used dictionary for word 'reference' and for zh i used google translator. I can remove wrong translation if necessary.

**Comment by KnorpelSenf** [comment]
Alrighty, please leave them in. I'll ping our translators to have a quick look, but I guess then we can merge :)

**Review by KnorpelSenf — APPROVED** [review]

File: site/docs/uk/hosting/supabase.md [hunk 1]
```diff
@@ -85,12 +85,12 @@ supabase secrets set BOT_TOKEN=123:aBcDeF-gh
...
 
 ```text
-https://api.telegram.org/bot<токен-бота>/setWebhook?url=https://<назва-проєкту>.functions.supabase.co/telegram-bot?secret=<токен-бота>
+https://api.telegram.org/bot<токен-бота>/setWebhook?url=https://<ідентифікатор-проєкту>.supabase.co/functions/v1/telegram-bot?secret=<токен-бота>
 ```
 
 Замініть `<токен-бота>` на свій реальний токен бота.
 Також замініть другий аргумент `<токен-бота>` на свій реальний токен бота.
-Замініть `<назва-проєкту>` на назву вашого проєкту Supabase.
+Замініть `<ідентифікатор-проєкту>` ідентифікатором вашого проекту Supabase.
```
Comment (niusia-ua) [inline site/docs/uk/hosting/supabase.md:93]:
```suggestion
Замініть `<ідентифікатор-проєкту>` ідентифікатором вашого проєкту Supabase.
```

**Review by niusia-ua — CHANGES_REQUESTED** [review]

**Review by niusia-ua — APPROVED** [review]

**Review by roj1512 — APPROVED** [review]
Thank you!

**Pull request merged**